list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'my_enumerable'
```

Comment:
- [ ] @Team great work on your list class, however, you have imported the module enumerable wrongly, it should be `enumerable` instead of `my_enumerable` since that is the name of your module file.

Reply:
Thanks for pointing that out @danmainah